cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,17 +1,44 @@
 #!/bin/sh
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z "${AWS_SHARED_CREDENTIALS_FILE}" ]; then echo "ENV VAR AWS_SHARED_CREDENTIALS_FILE is not set"; exit 1; fi
+
 ### expirationDate is always with timezone +00:00, eg, "2019-07-23T22:35+0000"
-MY_DATE=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
-declare -a regions=("us-east-1" "us-east-2" "us-west-1")
+my_date=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
+
+if [[ -n "${IPI_DEPROVISION_DATE-}" ]]; then
```

Comment:
Hmm sorry -- we will want this to be relative to the time that is run, right? So the config option would be how old --

```
$ date --date='1 hour ago'
Tue Jul 23 16:54:10 PDT 2019
```

Reply:
Done.